docs/tests/development/get_v1_messages/validation.md

```diff
@@ -0,0 +1,5 @@
+# Validation Tests
+
+These tests target the API endpoint GET /v1/messages testing 404 responses when invalid data is provided.
+
+
```

Comment:
Looks like there should be some content in here @Kennedy1265?

Reply:
Updated - also renamed to be in line with sandbox